### PR TITLE
[CP-966] I attach a template to a message.

### PR DESCRIPTION
**Pull request by wercix**
Jira: [CP-966]

**Description**
Adding template to message.

<details>
<summary><b>Screenshots</b></summary>
Screen Shot 2022-07-05 at 11 06 15

</details>

[CP-966]: https://appnroll.atlassian.net/browse/CP-966?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by wojcik92michal — APPROVED** [review]

File: packages/app/src/messages/components/templates-select-modal/templates-select-modal.component.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+/**
```
Comment (igorbezsmertnyi) [inline packages/app/src/messages/components/templates-select-modal/templates-select-modal.component.tsx:1]: I guess the `TemplatesSelectModal` is a part of the `templates` module, so let's move it.

File: packages/app/src/messages/components/templates-select-modal/templates-select-modal.component.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import { ModalSize } from "App/__deprecated__/renderer/components/core/modal/modal.interface"
+import { ModalDialog } from "App/ui/components/modal-dialog"
+import Table, {
+  Col,
+  Row,
+  RowSize,
+} from "App/__deprecated__/renderer/components/core/table/table.component"
+import { Template } from "App/templates/dto"
+import { TemplatesSelectModalTestIds } from "App/messages/components/templates-select-modal/templates-select-modal-test-ids.enum"
+export interface TemplatesSelectModalProps {
```
Comment (igorbezsmertnyi) [inline packages/app/src/messages/components/templates-select-modal/templates-select-modal.component.tsx:19]: Let's move it to separate file

File: packages/app/src/messages/components/templates-select-modal/templates-select-modal.component.tsx (hunk 1 above)
Comment (wercix) [reply packages/app/src/messages/components/templates-select-modal/templates-select-modal.component.tsx]: Moved in [7190298](https://github.com/mudita/mudita-center/pull/823/commits/719029829faf01967692767648a2d9a9ed576f54)

File: packages/app/src/messages/components/templates-select-modal/templates-select-modal.component.tsx (hunk 2 above)
Comment (wercix) [reply packages/app/src/messages/components/templates-select-modal/templates-select-modal.component.tsx]: Props moved in [75ca6c9](https://github.com/mudita/mudita-center/pull/823/commits/75ca6c9f6535fba4f0a5872a3c814e643025e196)

**Review by wojcik92michal — APPROVED** [review]

**Pull request merged**